mkosi/backend.py

```diff
@@ -35,6 +35,7 @@
     Union,
     cast,
 )
+from . import root_home
```

Comment:
Please move root_home from __init__.py to backend.py as well. We don't want imports from the main file into backend.py